elasticsearch/Dockerfile

```diff
@@ -27,14 +27,19 @@ RUN \
   rm -f $ES_PKG_NAME.tar.gz && \
   mv /$ES_PKG_NAME /elasticsearch
 
-
 VOLUME ["/data"]
+WORKDIR /data
 
 ADD elasticsearch.yml /elasticsearch/config/elasticsearch.yml
 
-WORKDIR /data
-
 EXPOSE 9200
 EXPOSE 9300
 
+# Retrieve github.com/anchor/elasticsearch-scripts and place them in /usr/local/sbin
+RUN \
+  cd / && \
+  wget https://github.com/anchor/elasticsearch-scripts/archive/v1.0.1.tar.gz -q \
+	-O es-scripts-1.0.1.tar.gz && \
+  tar xvzf es-scripts-1.0.1.tar.gz -C /usr/local/sbin --strip 1
```

Comment:
I'm not sure this will work (it won't install deps). Why not install `python2-pip` and `pip install elasticsearch-scripts` to grab everything from PyPI?